scripts/globals/chocobo_digging.lua

```diff
@@ -121,30 +121,30 @@ local digInfo =
     },
     [tpz.zone.WAJAOM_WOODLANDS] = -- 51
     {
-        {  770,  50, digReq.NONE    },
-        { 2150,  60, digReq.NONE    },
-        {  622, 197, digReq.NONE    },
-        { 2155,  23, digReq.NONE    },
-        {  739,   5, digReq.NONE    },
-        {17296, 133, digReq.NONE    },
-        {  703,  69, digReq.NONE    },
-        { 2213, 135, digReq.NONE    },
-        {  838,  21, digReq.NONE    },
+        {  646,   3, digReq.NONE    },
+        {  845,  75, digReq.NONE    },
+        {  770,  31, digReq.NONE    },
+        {  702,  52, digReq.NONE    },
+        {  833, 190, digReq.NONE    },
+        {17296, 147, digReq.NONE    },
+        { 2164,  17, digReq.NONE    },
+        { 2213, 152, digReq.NONE    },
+        {  838,   4, digReq.NONE    },
         { 4096, 100, digReq.NONE    }, -- all crystals
         { 1255,  10, digReq.NONE    }, -- all ores
-        {  688, 144, digReq.BURROW  },
-        {  702,  14, digReq.BURROW  },
-        {  690,  23, digReq.BURROW  },
-        { 1446,   3, digReq.BURROW  },
-        {  700,  14, digReq.BURROW  },
-        {  699,  37, digReq.BURROW  },
-        {  701,  28, digReq.BURROW  },
-        {  696,  23, digReq.BURROW  },
-        {  678,   9, digReq.BORE    },
-        {  645,   3, digReq.BORE    },
-        {  768, 193, digReq.BORE    },
+        {  688, 137, digReq.BURROW  },
+        {  690,  10, digReq.BURROW  },
+        { 1446,   1, digReq.BURROW  },
+        {  700,   4, digReq.BURROW  },
+        {  699,  31, digReq.BURROW  },
+        {  701,   4, digReq.BURROW  },
+        {  696,  35, digReq.BURROW  },
+        {  678,   1, digReq.BORE    },
+        {  645,   1, digReq.BORE    },
+        {  768,  98, digReq.BORE    },
         {  737,  22, digReq.BORE    },
         { 2475,   3, digReq.BORE    },
+		{  739,   1, digReq.BORE    },
```

Comment:
this tab indent can be converted into spaces

Reply:
Fixed, thanks for all the help with git gui as well!